k2/python/csrc/torch/v2/ragged_arc.cu

```diff
@@ -69,6 +69,19 @@ RaggedArc::RaggedArc(
   // TODO: we also need to pass the name of extra_labels and ragged_labels.
 }
 
+RaggedArc::RaggedArc(const Ragged<Arc> &fsa, torch::Tensor aux_labels)
+    : fsa(fsa) {
+  K2_CHECK_EQ(fsa.NumElements(), aux_labels.numel());
+  if (HasAttr("aux_labels")) DeleteAttr("aux_labels");
```

Comment:
You don't need to test it. The object is not fully initialized yet. Its attributes are empty.